inference.py

```diff
@@ -1,128 +1,111 @@
 import torch
 from transformers import AutoTokenizer, AutoModelForCausalLM
+import os
+from functionary_utils import SchemaGen
+from typing import List
+import bitsandbytes
 
-from schema import generate_schema_from_functions
+default_SYSTEM_MESSAGE = """A chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions. The assistant calls functions with appropriate input when necessary"""
```

Comment:
can we make this all capital case:
See:
https://peps.python.org/pep-0008/#constants